### PR TITLE
Add openstack image min_disk option

### PR DESCRIPTION
**Pull request by y-kuno**
Adds an option to `min-disk` of the build image to the OpenStack.

### CODE REVIEW

**Comment by rickard-von-essen** [comment]
Thanks for contributing this. I'll try to review this soon, but int the meantime it would be great if you could look into the tests. `make test` should pass.

**Comment by y-kuno** [comment]
Passed make test with this commit.
Please p-r review.

**Comment by rickard-von-essen** [comment]
Unfortunately I'll not have any time to test this during the comming two weeks. But I agree that the docs should be a bit more expensive.

File: website/source/docs/builders/openstack.html.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ builder.
     after creation. An image member is usually a project (also called the
     "tenant") with whom the image is shared.
 
+-   `image_min_disk` (int) - Minimum disk of needed to boot image size in GB.
```
Comment (rickard-von-essen) [inline website/source/docs/builders/openstack.html.md]: Maybe this from the cli docs:

_Minimum disk size needed to boot image, in gigabytes_

**Comment by SwampDragons** [comment]
What's the status on this one?  Looks like we have some requested changes that have not been made. Should we close the PR, or does someone want to move forward with getting it ready for merge?

@rickard-von-essen @y-kuno

**Comment by rickard-von-essen** [comment]
@SwampDragons it's two comments regarding wording, maybe you can apply those changes and merge it?

**Pull request merged**